api_views/users.py

```diff
@@ -12,6 +12,15 @@
 def error_message_helper(msg):
     return '{ "status": "fail", "message": "' + msg + '"}'
 
+def error_message_helper1(msg):
+    try:
+        jsonschema.validate(msg, msg)
+    except:
+        return {"status": 200, "message": "' + msg + '"}
```

Comment:
Status code 200 used for error response is misleading - should use appropriate error code